app/controllers/api/v1/search_results_controller.rb

```diff
@@ -19,8 +14,28 @@ def create
         end
       end
 
+      def index
+        search_result_list = SearchResult.where(user_id: current_user.id).order(:id)
```

Comment:
Not needed now but the team usually uses https://github.com/varvet/pundit to automatically filter and authorize current user actions 💡